lib/gems/pending/appliance_console.rb

```diff
@@ -150,37 +150,44 @@ module ApplianceConsole
       clear_screen
       selection = ask_with_menu("Advanced Setting", AS_OPTIONS, nil, true)
       case selection
-      when I18n.t("advanced_settings.dhcp")
-        say("DHCP Network Configuration\n\n")
-        if agree("Apply DHCP network configuration? (Y/N): ")
-          say("\nApplying DHCP network configuration...")
-
-          resolv = LinuxAdmin::Dns.new
-          resolv.search_order = []
-          resolv.nameservers = []
-          resolv.save
+      when I18n.t('advanced_settings.networking')
+        options = {
+          'Set DHCP Network Configuration'   => 'dhcp',
+          'Set Static Network Configuration' => 'static',
+          'Test Network Configuration'       => 'testnet',
+          'Set Hostname'                     => 'hostname'
```

Comment:
When working on this, 13 days ago, I contacted our i18n guy and he was ok with dropping few sentences from the yaml given the overall state of the localization of the appliance_console.

Majority (!) of the stuff in the appliance_console is not localized. Even the stuff that is localized is not translated.

We can work on the localization as a separate issue. Perhaps we could start after the plans from the UX team materialize. That way, we will be sure to translate final strings.